apm/docs/metrics.md

```diff
@@ -671,6 +673,15 @@ Consensus time for P90
 ```
 avg(histogram_quantile(0.90, sum(rate(muta_consensus_duration_seconds_bucket[5m])) by (le, instance)))
 ```
+
+##### /
+Not show, for alert
+```
+avg(histogram_quantile(0.90, sum(rate(muta_consensus_duration_seconds_bucket[5m])) by (le, instance))) / avg(histogram_quantile(0.90, sum(rate(muta_consensus_time_cost_seconds_bucket{type="exec"}[5m])) by (le, instance))) 
```

Comment:
Would you explain what this expression does?

Reply:
Used to determine if the consensus time is greater than the exec time

Reply:
So it alerts when the ratio exceeds 1, right?  It would be better to mark this result in the document

Reply:
Yeah, this is going to move into the alert table file.